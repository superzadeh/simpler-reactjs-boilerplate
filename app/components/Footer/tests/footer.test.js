import React from 'react';
import Footer from '../footer';

describe('Footer', () => {
  it('should render correctly', () => {
    const footer = (<Footer></Footer>);
    expect(footer).toMatchSnapshot();
  });
});
