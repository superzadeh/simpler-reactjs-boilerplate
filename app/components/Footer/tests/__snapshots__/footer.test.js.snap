exports[`Footer should render correctly 1`] = `<Footer />`;
